db/fixtures/server_roles.csv

```diff
@@ -1,5 +1,6 @@
 name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
+cockpit_ws,Cockpit,1,false,region
```

Comment:
The idea is that there is one cockpit-ws running per domain used in the browser. The only port that needs to be open for this setup to work is ssh. Is that not workable? What ports are usually reachable between zones?